akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -129,22 +130,14 @@ private[akka] object Shard {
           entityProps,
           settings,
           extractEntityId,
-          extractShardId,
           handOffStopMessage,
           replicator,
           majorityMinCap)).withDeploy(Deploy.local)
     } else if (settings.rememberEntities && settings.stateStoreMode == ClusterShardingSettings.StateStoreModePersistence)
-      Props(
-        new PersistentShard(
-          typeName,
-          shardId,
-          entityProps,
-          settings,
-          extractEntityId,
-          extractShardId,
-          handOffStopMessage)).withDeploy(Deploy.local)
+      Props(new PersistentShard(typeName, shardId, entityProps, settings, extractEntityId, handOffStopMessage))
```

Comment:
So the scalafmt sbt plugin leaves this be, but the autoformatter puts it all on one line anyway? Weird.